### PR TITLE
feat: add ci pipeline for testing Express Mongoose

### PR DESCRIPTION
**Pull request by Abbhiishek**
## What does this PR do?

This PR adds GitHub Action Test Bench for Repo https://github.com/keploy/samples-typescript/tree/main/express-mongoose

## Related PRs and Issues

Closes: [#2199](https://github.com/keploy/keploy/issues/2199)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] Have you read the [Contributing Guidelines on issues](https://keploy.io/docs/keploy-explained/contribution-guide/)?
- [x] My code follows the style guidelines of this project.

### CODE REVIEW

**Comment by Abbhiishek** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Abbhiishek** [comment]
recheck

**Review by shivamsouravjha — COMMENTED** [review]
the test bench fails~

Please refer to this action log. https://github.com/keploy/keploy/actions/runs/10347804490/job/28649173467?pr=2200

File: .github/workflows/coverage_stage.yml [hunk 1]
```diff
@@ -20,16 +20,30 @@ jobs:
...
           path: ./temp-coverage-dir
 
-      - name: Download second coverage data of gin-mongo application 
+      - name: Download second coverage data of gin-mongo application
         uses: actions/download-artifact@v4
         with:
           name: coverage-reports-samples-go-gin-mongo-1
           path: ./temp-coverage-dir
 
+      - name: Download second coverage data of exporess-mongoose application
```
Comment (gouravkrosx) [inline .github/workflows/coverage_stage.yml:29]: fix the typo `exporess` and below as well.

File: .github/workflows/test-express-mongoose-2.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        run: |
+          echo "You can add the noisy fields in the config file if any."
+
+      - name: Run testing script for express-mongoose application
+        uses: ./.github/actions/tester
+        with:
+          working-directory: samples-typescript/express-mongoose
+          command: node src/app.js
+          delay: 7
+          keploy-record-bin: /usr/local/bin/keployR
```
Comment (gouravkrosx) [inline .github/workflows/test-express-mongoose-2.yml:56]: you need to change the record and test binary, it should be `keploy-record-bin: /usr/local/bin/keployB`, `keploy-test-bin:/usr/local/bin/keployR`

**Comment by Abbhiishek** [comment]
Refer to https://github.com/keploy/keploy/pull/2217

**Pull request closed without merging**